packages/terra-clinical-alert/tests/jest/Alert.test.jsx

```diff
@@ -0,0 +1,143 @@
+import React from 'react';
+/* eslint-disable import/no-extraneous-dependencies */
+import { IntlProvider } from 'react-intl';
+
```

Comment:
Those were for a sanity check, testing that each of the props get passed through to the terra-alert appropriately. I can remove them if you don't think it's needed.